site/docs/v1/tech/troubleshooting.adoc

```diff
@@ -113,12 +114,24 @@ When the above modules are available to your Java runtime, configure JMX by pass
 
 include::docs/v1/tech/email-templates/_email-troubleshooting.adoc[]
 
-== Troubleshooting SAML or OIDC Identity Provider Configuration
+== Troubleshooting Identity Provider Configuration
 
 :idp_type_name: SAML or OIDC
 include::docs/v1/tech/identity-providers/_idp-managed-domains-troubleshooting.adoc[]
 :idp_type_name!:
 
+=== SAML Identity Provider
+
+SAML Identity Providers will not work without the proper CORS configuration. 
+
+If you are setting up a SAML Identity Provider in FusionAuth, link:/docs/v1/tech/identity-providers/samlv2/#cors-configuration[ensure you have configured CORS correctly].
+
+To see if CORS is the issue preventing SAML functionality from working:
+
+* Enable debugging by navigating to [breadcrumb]#Settings -> System -> CORS# and selecting [field]#Debug enabled# and saving the settings.
+* Attempt to log in again using the provider.
+* Review the Event Log for relevant messages. See [breadcrumb]#System -> Event Log# to view this log.
+
```

Comment:
This is great. 

I would mention that the "tell tale sign" is a a 403 status code. This can be viewed in the browser network console. 

And if they enable CORS debug, ensure they turn it back off when they are done - otherwise this could get really spammy.